changelog/unreleased/issue-1800

```diff
@@ -0,0 +1,10 @@
+Bugfix: omit ENOTDATA for extended attributes
```

Comment:
That sounds rather technical and it is not clear which command is affected. What about
`Bugfix: Ignore 'no data available' error during backup`